In [2]:
combination = [(1,1,1,1,1,1),
(1,1,1,1,1,0),
(1,1,1,1,0,1),
(1,1,1,1,0,0),
(1,1,1,0,1,1),
(1,1,1,0,1,0),
(1,1,1,0,0,1),
(1,1,1,0,0,0),
(1,1,0,1,1,1),
(1,1,0,1,1,0),
(1,1,0,1,0,1),
(1,1,0,1,0,0),
(1,1,0,0,1,1),
(1,1,0,0,1,0),
(1,1,0,0,0,1),
(1,1,0,0,0,0),
(1,0,1,1,1,1),
(1,0,1,1,1,0),
(1,0,1,1,0,1),
(1,0,1,1,0,0),
(1,0,1,0,1,1),
(1,0,1,0,1,0),
(1,0,1,0,0,1),
(1,0,1,0,0,0),
(1,0,0,1,1,1),
(1,0,0,1,1,0),
(1,0,0,1,0,1),
(1,0,0,1,0,0),
(1,0,0,0,1,1),
(1,0,0,0,1,0),
(1,0,0,0,0,1),
(1,0,0,0,0,0),
(0,1,1,1,1,1),
(0,1,1,1,1,0),
(0,1,1,1,0,1),
(0,1,1,1,0,0),
(0,1,1,0,1,1),
(0,1,1,0,1,0),
(0,1,1,0,0,1),
(0,1,1,0,0,0),
(0,1,0,1,1,1),
(0,1,0,1,1,0),
(0,1,0,1,0,1),
(0,1,0,1,0,0),
(0,1,0,0,1,1),
(0,1,0,0,1,0),
(0,1,0,0,0,1),
(0,1,0,0,0,0),
(0,0,1,1,1,1),
(0,0,1,1,1,0),
(0,0,1,1,0,1),
(0,0,1,1,0,0),
(0,0,1,0,1,1),
(0,0,1,0,1,0),
(0,0,1,0,0,1),
(0,0,1,0,0,0),
(0,0,0,1,1,1),
(0,0,0,1,1,0),
(0,0,0,1,0,1),
(0,0,0,1,0,0),
(0,0,0,0,1,1),
(0,0,0,0,1,0),
(0,0,0,0,0,1),
(0,0,0,0,0,0)]

weights = [0.1521, 0.1522, 0.1013, 0.1522, 0.2511, 0.1910]

In [3]:
for comb in combination:
    tot = 0
    for idx, val in enumerate(comb):
        # print(val, weights[idx])
        tot = tot + val*weights[idx]
    truth = list()
    for i in comb:
        if i == 1:
            truth.append("true")
        else:
            truth.append("false")
    print(tuple(truth), round(tot,2), ",", round(1-tot,2), ";")

('true', 'true', 'true', 'true', 'true', 'true') 1.0 , 0.0 ;
('true', 'true', 'true', 'true', 'true', 'false') 0.81 , 0.19 ;
('true', 'true', 'true', 'true', 'false', 'true') 0.75 , 0.25 ;
('true', 'true', 'true', 'true', 'false', 'false') 0.56 , 0.44 ;
('true', 'true', 'true', 'false', 'true', 'true') 0.85 , 0.15 ;
('true', 'true', 'true', 'false', 'true', 'false') 0.66 , 0.34 ;
('true', 'true', 'true', 'false', 'false', 'true') 0.6 , 0.4 ;
('true', 'true', 'true', 'false', 'false', 'false') 0.41 , 0.59 ;
('true', 'true', 'false', 'true', 'true', 'true') 0.9 , 0.1 ;
('true', 'true', 'false', 'true', 'true', 'false') 0.71 , 0.29 ;
('true', 'true', 'false', 'true', 'false', 'true') 0.65 , 0.35 ;
('true', 'true', 'false', 'true', 'false', 'false') 0.46 , 0.54 ;
('true', 'true', 'false', 'false', 'true', 'true') 0.75 , 0.25 ;
('true', 'true', 'false', 'false', 'true', 'false') 0.56 , 0.44 ;
('true', 'true', 'false', 'false', 'false', 'true') 0.5 , 0.5 ;
('true', 'true', 'false', 'false', 

(true, true, true, true, true, false) 0.81 , 0.19 ;
(true, true, true, true, false, true) 0.75 , 0.25 ;
(true, true, true, true, false, false) 0.56 , 0.44 ;
(true, true, true, false, true, true) 0.85 , 0.15 ;
(true, true, true, false, true, false) 0.66 , 0.34 ;
(true, true, true, false, false, true) 0.6 , 0.4 ;
(true, true, true, false, false, false) 0.41 , 0.59 ;
(true, true, false, true, true, true) 0.9 , 0.1 ;
(true, true, false, true, true, false) 0.71 , 0.29 ;
(true, true, false, true, false, true) 0.65 , 0.35 ;
(true, true, false, true, false, false) 0.46 , 0.54 ;
(true, true, false, false, true, true) 0.75 , 0.25 ;
(true, true, false, false, true, false) 0.56 , 0.44 ;
(true, true, false, false, false, true) 0.5 , 0.5 ;
(true, true, false, false, false, false) 0.3 , 0.7 ;
(true, false, true, true, true, true) 0.85 , 0.15 ;
(true, false, true, true, true, false) 0.66 , 0.34 ;
(true, false, true, true, false, true) 0.6 , 0.4 ;
(true, false, true, true, false, false) 0.41 , 0.59 ;
(true, false, true, false, true, true) 0.7 , 0.3 ;
(true, false, true, false, true, false) 0.5 , 0.5 ;
(true, false, true, false, false, true) 0.44 , 0.56 ;
(true, false, true, false, false, false) 0.25 , 0.75 ;
(true, false, false, true, true, true) 0.75 , 0.25 ;
(true, false, false, true, true, false) 0.56 , 0.44 ;
(true, false, false, true, false, true) 0.5 , 0.5 ;
(true, false, false, true, false, false) 0.3 , 0.7 ;
(true, false, false, false, true, true) 0.59 , 0.41 ;
(true, false, false, false, true, false) 0.4 , 0.6 ;
(true, false, false, false, false, true) 0.34 , 0.66 ;
(true, false, false, false, false, false) 0.15 , 0.85 ;
(false, true, true, true, true, true) 0.85 , 0.15 ;
(false, true, true, true, true, false) 0.66 , 0.34 ;
(false, true, true, true, false, true) 0.6 , 0.4 ;
(false, true, true, true, false, false) 0.41 , 0.59 ;
(false, true, true, false, true, true) 0.7 , 0.3 ;
(false, true, true, false, true, false) 0.5 , 0.5 ;
(false, true, true, false, false, true) 0.44 , 0.56 ;
(false, true, true, false, false, false) 0.25 , 0.75 ;
(false, true, false, true, true, true) 0.75 , 0.25 ;
(false, true, false, true, true, false) 0.56 , 0.44 ;
(false, true, false, true, false, true) 0.5 , 0.5 ;
(false, true, false, true, false, false) 0.3 , 0.7 ;
(false, true, false, false, true, true) 0.59 , 0.41 ;
(false, true, false, false, true, false) 0.4 , 0.6 ;
(false, true, false, false, false, true) 0.34 , 0.66 ;
(false, true, false, false, false, false) 0.15 , 0.85 ;
(false, false, true, true, true, true) 0.7 , 0.3 ;
(false, false, true, true, true, false) 0.5 , 0.5 ;
(false, false, true, true, false, true) 0.44 , 0.56 ;
(false, false, true, true, false, false) 0.25 , 0.75 ;
(false, false, true, false, true, true) 0.54 , 0.46 ;
(false, false, true, false, true, false) 0.35 , 0.65 ;
(false, false, true, false, false, true) 0.29 , 0.71 ;
(false, false, true, false, false, false) 0.1 , 0.9 ;
(false, false, false, true, true, true) 0.59 , 0.41 ;
(false, false, false, true, true, false) 0.4 , 0.6 ;
(false, false, false, true, false, true) 0.34 , 0.66 ;
(false, false, false, true, false, false) 0.15 , 0.85 ;
(false, false, false, false, true, true) 0.44 , 0.56 ;
(false, false, false, false, true, false) 0.25 , 0.75 ;
(false, false, false, false, false, true) 0.19 , 0.81 ;